pkgs/by-name/pl/playdate-sdk/package.nix

```
@@ -0,0 +1,86 @@
+{
+  autoPatchelfHook,
+  fetchurl,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  pkgs,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "playdate-sdk";
+  version = "2.5.0";
+
+  src = fetchurl {
+    url = "https://download.panic.com/playdate_sdk/Linux/PlaydateSDK-${version}.tar.gz";
+    sha256 = "sha256-1b7j7lkN16YO4EUWyZPZ+PPC9Sa3AFoN5c84ArTGXok=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  buildInputs = with pkgs; [ webkitgtk ];
```

```suggestion
  buildInputs = [ webkitgtk_4_0 ];
```
Or another version of webkitgtk